toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -661,3 +661,51 @@ func extractWorkerTar(chroot string, workerTar string) (err error) {
 	_, _, err = shell.Execute("tar", "-I", gzipTool, "-xf", workerTar, "-C", chroot)
 	return
 }
+
+type ChrootInterface interface {
```

Comment:
I would place `ChrootInterface` and `DummyChroot` in their own files (but still within this package/directory).